tox.ini

```diff
@@ -1,39 +1,36 @@
 [tox]
-envlist = py
+envlist = py26,py27
```

Comment:
heh, not only does it need updating, it has a typo -- the environment variable is TOXENV